tests/integration/suite.go

```diff
@@ -42,10 +42,6 @@ func (s *integrationTestSuite) Logf(format string, args ...any) {
 	s.T().Logf(format, args...)
 }
 
-func (s *integrationTestSuite) Run(name string, subtestFunc func(t *testing.T)) bool {
-	return s.T().Run(name, subtestFunc)
-}
-
```

Comment:
This didn't seem to be needed but please correct me if there was a reason to wrap this